im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,426 @@
+      <label form="short" prefix=" (" suffix=")" text-case="capitalize-first"/>
+      <substitute>
+        <names variable="editor" suffix=": "/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" delimiter="/" delimiter-precedes-last="always" et-al-min="4" et-al-use-first="2" initialize-with=". "/>
+      <substitute>
+        <names variable="editor">
+          <name form="short" delimiter="/" delimiter-precedes-last="always" et-al-min="4" et-al-subsequent-min="2"/>
+        </names>
+        <names variable="collection-editor">
+          <name form="short" et-al-min="4" et-al-subsequent-min="2"/>
+        </names>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="URL">
+        <group>
+          <text variable="URL"/>
+          <group prefix=" [" suffix="]">
+            <date variable="accessed">
+              <date-part name="day" form="numeric-leading-zeros" suffix="."/>
+              <date-part name="month" form="numeric-leading-zeros" suffix="."/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group>
+      <text variable="publisher"/>
+    </group>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date date-parts="year" form="text" variable="issued" prefix=" ">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper broadcast thesis webpage chapter paper-conference" match="any">
+        <label strip-periods="true" prefix=", " suffix=". " variable="page" form="short"/>
+        <text variable="page"/>
+        <text variable="note" prefix=" [erstmals " suffix="]"/>
+      </if>
+      <else-if type="entry-encyclopedia" match="any">
+        <label variable="locator" form="short" suffix=". " strip-periods="true"/>
+        <text variable="page"/>
+        <text variable="note" prefix=" [erstmals " suffix="]"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="citation-locator">
+    <choose>
+      <if type="manuscript" match="none">
+        <label variable="locator" form="short" suffix=". " strip-periods="true"/>
+      </if>
+    </choose>
+    <text variable="locator"/>
+  </macro>
+  <macro name="date">
+    <choose>
+      <if match="any" variable="issued">
+        <date form="text" variable="issued"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="collection-editor">
+    <names variable="collection-editor" prefix="Hrsg. von ">
+      <name delimiter="/" delimiter-precedes-last="always"/>
```

Comment:
I think this would be a good idea in most cases. In those cases however were there is no author but an editor it would produce strange results. By now in such a case the result would be "Wolfgang Röllig (Hrsg.): Das Gilgamesch-Epos. (Reclam-Bibliothek, Bd. 10702) Stuttgart: Reclam 2009.". If I treat "Hrsg. von" as <label form="verb-short"/> the result would be "Wolfgang Röllig (Hrsg. von): Das Gilgamesch-Epos. (Reclam-Bibliothek, Bd. 10702) Stuttgart: Reclam 2009.", wouldn't it?